keadm/app/cmd/util/ubuntuinstaller.go

```diff
@@ -294,6 +296,14 @@ func (u *UbuntuOS) addK8SRepositoryAndUpdate() error {
 		return fmt.Errorf("ubuntu dist version not available")
 	}
 	fmt.Println("Ubuntu distribution version is", distVersion)
+	distVersionForSuite := distVersion
+	if distVersion == "bionic" {
```

Comment:
Define "bionic" and "xenial" as constants and use it here.

Reply:
done